asapdiscovery-ml/asapdiscovery/ml/tests/test_ml_cli.py

```diff
@@ -392,8 +393,8 @@ def test_build_trainer_visnet(exp_file, docked_files, tmp_path):
             tmp_path / "ds_cache.pkl",
             "--ds-config-cache",
             tmp_path / "ds_config_cache.json",
-            "--loss-type",
-            "mse_step",
+            "--loss",
+            "loss_type:mse_step",
```

Comment:
Is the default still preserved wrt old CLI?

Reply:
I don't think there were any defaults before? you've always had to specify which loss function you want to use